conventions/documentation/primarynavigation.md

```diff
@@ -1,13 +1,14 @@
 ---
-parent: Conventions
-version: 4.2
+name: Primary navigation
+menu: Navigation
 ---
 
 # Primary navigation
```

Comment:
IIRC the interactions described in this page as a whole may not meet our [current (but still rough) definition](https://docs.google.com/presentation/d/11Ig7RUSIZZJIvvAW_lmxwLOLVvPSf2HfGHPyUAIGxEw/edit#slide=id.g6eb3dfcc6f_2_49) of a "convention". Do we think this page has day-to-day value as a reference for designers working on new flows?

Reply:
@andybraren is this a question you want to visit in this PR?

Reply:
If we agree to remove this page, that could be done in this variety pack PR or a future one. I don't consider this a blocker for this particular PR.